packages/shared/src/components/buttons/Button.tsx

```diff
@@ -95,13 +95,16 @@ function ButtonComponent<TagName extends AllowedTags>(
         variant !== ButtonVariant.Option && 'justify-center font-bold',
         { iconOnly },
         iconOnly ? IconOnlySizeToClassName[size] : SizeToClassName[size],
+        iconPosition === ButtonIconPosition.Top && `flex-col`,
         !color && VariantToClassName[variant],
         VariantColorToClassName[variant]?.[color],
         className,
       )}
     >
       {icon &&
-        iconPosition === ButtonIconPosition.Left &&
+        [ButtonIconPosition.Left, ButtonIconPosition.Top].includes(
```

Comment:
If you don't want to instantiate each time youcan pull array as const above component.